app/models/library_entry.rb

```diff
@@ -149,12 +149,9 @@ def activity
   end
 
   def kind
```

Comment:
Explosions, probably.  But it's something that should never happen, so I'm not terribly worried about that